test/test_unary_ufuncs.py

```diff
@@ -1219,6 +1219,26 @@ def check_equal(t):
         t = torch.tensor([min, max, eps, tiny], dtype=dtype, device=device)
         check_equal(t)
 
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
+    def test_special_ndtr_vs_scipy(self, device, dtype):
```

Comment:
Do we really need this test?

Reply:
I think it is okay to have this test for now.

I did try previously to add `torch.finfo(eps), torch.finfo(max), etc` to reference numerics but few tests failed.

So, as a follow-up to `i1`, `i1e` and this PR, I will update the reference numerics (after syncing with @pmeier) to extend the range of `make_tensor` and also add `eps, tiny, min, max` from finfo to `reference_numerics_extreme` .

Previous conversation: https://github.com/pytorch/pytorch/pull/56352#discussion_r633208717